neurovault/apps/statmaps/models.py

```diff
@@ -26,6 +26,11 @@
 from neurovault.settings import PRIVATE_MEDIA_ROOT
 
 
+# I wanted to put this in utils, but it had to go here to avoid circular imports. :-/
+### I think it should probably be read from a template object? 
+def get_target_template_list():
+    return [('MNI152', 'Human'), ('NMT', 'Rhesus (macacca mulatta)'), ('Dorr_2008_average', 'Mouse (Dorr et al., 2008 space)')]
```

Comment:
`'MNI152NLin2009cAsym', 'Human (MNI152 NLin 2009c Asym)'`

Reply:
Is this another template? I'll add it, but you'll need to share the template with me. :-)

Reply:
nope - it's the more accurate description of the current template